### PR TITLE
Fix header and footer pattern page accessibility issues

### LINKED ISSUE #283: Header and footer page accessibility issues
**Describe the bug**
Component examples on the Header and footer page are the same as the ones being used globally on the site and are therefore causing AXE to flag the following issues:

- **Heading levels should only increase by one** - caused by `h1` used in the 'With title variant in header' example
- **Ensure landmarks are unique** - caused by header navigation example matching the site's navigation

[https://designsystem.wmnetwork.co.uk/patterns/header-and-footer/](https://designsystem.wmnetwork.co.uk/patterns/header-and-footer/)

### PR DESCRIPTION
**Pull request by gldgrnt**
Managed to fix the issues that AXE and IBM flag for duplicate `<nav>` components. Not sure how to fix the issue with the `h1` in the 'Header with title' as the `h1` needs to be present in the code example. Fixes #283

### CODE REVIEW

File: src/www/data.njk.json [hunk 1]
```diff
@@ -38,6 +38,9 @@
         {
           "name": "Disruption indicators"
         },
+        {
+          "name": "File download"
+        },
```
Comment (catiarodriguescosta) [inline src/www/data.njk.json:43]: Why is this in this PR? Isn't it already merged?

File: src/www/pages/components/file-download/index.njk [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    <li>When the file cannot be made accessible. For example, complex maps.</li>
+</ul>
+{# When not to #}
+<h3>When not to use it?</h3>
+<ul>
+    <li>If the document is accessible to users with assistive technology. In that instance, use the accessible file component.</li>
+    <li>You still need to make the document accessible as soon as possible. Ideally before publishing it.</li>
+</ul>
+
+{% endblock %}
```
Comment (catiarodriguescosta) [inline src/www/pages/components/file-download/index.njk:50]: Why is this in this PR? Isn't it already merged?

**Review by catiarodriguescosta — CHANGES_REQUESTED** [review]
I left some comments. Have a look please. :)

**Comment by gldgrnt** [comment]
@catiarodriguescosta No sure why the `file-download` component things are appearing here. I've reset my master branch now

**Review by catiarodriguescosta — APPROVED** [review]
Looks good!

**Review by houbly — APPROVED** [review]

**Pull request merged**